server/cluster/cluster.go

```diff
@@ -246,27 +296,57 @@ func (c *Cluster) CreateSchema(ctx context.Context, schemaName string, schemaID
 	return schema, nil
 }
 
-func (c *Cluster) CreateTable(ctx context.Context, schemaName string, shardID uint32,
-	tableName string, tableID uint64,
-) (*Table, error) {
+func (c *Cluster) CreateTable(ctx context.Context, nodeName string, schemaName string, tableName string) (*Table, error) {
 	c.lock.Lock()
```

Comment:
Maybe read lock is enough to check whether the table exists.

Reply:
When we use read locks, an error occurs when multiple threads create tables with the same name